adapters/platform_win32.c

```diff
@@ -20,6 +20,9 @@
 #if USE_WOLFSSL
 #include "azure_c_shared_utility/tlsio_wolfssl.h"
 #endif
+#if USE_MBEDTLS
```

Comment:
ideally should these all be `#if` and `#elif`?

Reply:
We could. But the effect would be similar anyway if only one define is provided. 
On the other side, if more than one is provided, we better try to include everything and let the compilation fail, so customers know right away what is wrong.